source/common/router/router_ratelimit.cc

```diff
@@ -237,8 +274,14 @@ RateLimitPolicyEntryImpl::RateLimitPolicyEntryImpl(
       auto* factory = Envoy::Config::Utility::getFactory<RateLimit::DescriptorProducerFactory>(
           action.extension());
       if (!factory) {
-        throw EnvoyException(
-            absl::StrCat("Rate limit descriptor extension not found: ", action.extension().name()));
+        RateLimitDescriptorValidationVisitor validation_visitor;
```

Comment:
Yes, this is a fallback so it should work the same as before whenever an extension was found. I added a comment about conflicts on names and types.

I actually don't think there is a high chance of conflicts since this is a fairly new extension category. I would probably put more weight on using the standard input function category due to higher bang-for-the-buck.

One major difference is that rate limiting descriptors are three-valued while input functions are just optionals:
- delete the whole descriptor;
- add an entry;
- don't add entry but don't delete the descriptor.

I think in this case we can treat `Some("")` as case (3). Let me push an update to clarify this distincion.